llvm/lib/Target/RISCV/RISCVSubtarget.h

```diff
@@ -196,6 +196,9 @@ class RISCVSubtarget : public RISCVGenSubtargetInfo {
   bool hasCPOPLike() const {
     return HasStdExtZbb || (HasVendorXCVbitmanip && !IsRV64);
   }
+  bool hasREV8Like() const {
+    return HasStdExtZbb | HasStdExtZbkb || HasVendorXTHeadBb;
```

Comment:
Did you intend to mix arithmetic and short-circuiting or?

Reply:
No I did not. Thanks